frontend/src/app/features/work-packages/routing/partitioned-query-space-page/partitioned-query-space-page.component.sass

```diff
@@ -42,6 +42,10 @@
   .work-packages-partitioned-page--content-left
     flex: 1
 
+  // Move to the left so that the actual header text is aligned with the content below
+  .title-container_editable
```

Comment:
I don't think this is the right place for this CSS, I'd put it back where it was. I think a good solution is to make the `_editable` attribute a prop on the editable toolbar title component, and then setting it to `true` only where we need it.

Reply:
As discussed, we will merge this for now as we could not find any other places where this needs to be adapted.